index.html

```diff
@@ -210,22 +210,21 @@ <h3>New Technical Reports</h3>
     </dd>
   </dl>
 
-  <p>The Working Group MAY develop additional Recommendation-track and non-Recommendation-track Technical
-  Reports.</p>
+  <p>The Working Group MAY develop additional Recommendation-track and non-Recommendation-track specifications.</p>
 </div>
 
 <div id="revisions">
-  <h3>Existing Technical Reports</h3>
+  <h3>Existing Normative Specifications</h3>
```

Comment:
Maybe "Existing Technical Reports (including Normative Specifications)" to make sure that we include WG Notes etc.

Reply:
Actually, in the current template (see below), documents are separated by normative or non-normative, so I'd want to make items separated in our charter.
https://w3c.github.io/charter-drafts/charter-template.html#deliverables

Reply:
OK that can work too.